### PR TITLE
Support Traditional Chinese

### PR DESCRIPTION
**Pull request by antoncoding**
I just added Traditional Chinese support. I also updated a typo in the Simplified Chinese translation. Please help me verify the content if you can 😄

### CODE REVIEW

**Comment by clementbalestrat** [comment]
@antoncoding thanks! Will review and merge it shortly :)

File: src/components/Dropdown/LanguageDropdown.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import OutsideClickHandler from 'react-outside-click-handler';
 import { PLarge } from '../Typography';
 import i18n from 'i18next';
 
-const SUPPORTED_LANGUAGES = ['en', 'es', 'ru', 'zh'];
+const SUPPORTED_LANGUAGES = ['en', 'es', 'ru', 'zh', 'zh-tw'];
```
Comment (0dayZh) [inline src/components/Dropdown/LanguageDropdown.js]: Better to use 'zh-CN' for simplified Chinese.

ref: https://en.wikipedia.org/wiki/Language_localisation

File: src/components/Dropdown/LanguageDropdown.js [hunk 2]
```diff
@@ -19,6 +19,8 @@ const codeToLang = code => {
 			return 'Russian';
 		case 'zh':
 			return 'Chinese';
```
Comment (0dayZh) [inline src/components/Dropdown/LanguageDropdown.js]: `zh` -> `zh-CN`

```
return 'Chinese';
```

to 

```
return 'Chinese(simplified)
```

**Comment by clementbalestrat** [comment]
Thanks @antoncoding @0dayZh! Merging now.

**Pull request merged**